README.md

```diff
@@ -0,0 +1,15 @@
+# Aware
+![icon_256x256](https://cloud.githubusercontent.com/assets/896475/12147215/c598f528-b465-11e5-8b69-02b5cf823f06.png)
+
+Aware is a menubar app for OSX that displays how long you've been actively using your computer.
+
+## Installing the app
+
+1. To install Aware, begin by [downloading the latest release](https://github.com/josh/Aware/releases/latest/). 
+2. Unzip the package
+3. Move `Aware.app` to your `Applications` folder.
```

Comment:
Disregard all that. I think I used my Apple signed developer cert. So as long as its allowed by Gatekeeper, it should open just fine.